src/views/product-integration/component-view/server.ts

```diff
@@ -0,0 +1,180 @@
+// Third-party imports
```

Comment:
I like the location of this file here.  Much easier to navigate than what we had before.